lib/nostrum/struct/guild/audit_log.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Nostrum.Struct.Guild.AuditLog do
+  @moduledoc """
+  Represents a guild's audit log.
+  """
+
+  alias Nostrum.Struct.Guild.AuditLogEntry
+  alias Nostrum.Struct.{User, Webhook}
+  alias Nostrum.Util
+
+  defstruct [:entries, :users, :webhooks]
+
+  @typedoc "Entries of this guild's audit log."
```

Comment:
We don't typically have periods at the end of the typedoc definitions

Reply:
Removed in 44473a6